.github/workflows/CI.yaml

```
@@ -169,6 +169,19 @@ jobs:
         env:
           RUSTFLAGS: -Cinstrument-coverage
 
+      - name: "Test (regressions): pallets"
```

```suggestion
      - name: "Test (performance): pallets"
```
Why only regressions? 😢 
Here and in other places.